CondFormats/CTPPSReadoutObjects/plugins/TotemDAQMappingESSourceXML.cc

```diff
@@ -291,6 +336,22 @@ edm::ESProducts< boost::shared_ptr<TotemDAQMapping>, boost::shared_ptr<TotemAnal
 
 //----------------------------------------------------------------------------------------------------
 
+DOMDocument* TotemDAQMappingESSourceXML::GetDOMDocument(string file)
+{
+  XercesDOMParser* parser = new XercesDOMParser();
```

Comment:
OK, now parser is also deleted:
https://github.com/CTPPS/cmssw/blob/ctpps_conf_daq_mapping/CondFormats/CTPPSReadoutObjects/plugins/TotemDAQMappingESSourceXML.cc#L339
If you like this solution, I will backport it also to 8_0_X. Please let me know. Thanks!